rsocket-core/src/main/java/io/rsocket/RSocketClient.java

```diff
@@ -40,8 +46,8 @@
   private final Function<Frame, ? extends Payload> frameDecoder;
   private final Consumer<Throwable> errorConsumer;
   private final StreamIdSupplier streamIdSupplier;
-  private final NonBlockingHashMapLong<LimitableRequestPublisher> senders;
```

Comment:
I think handleSendProcessorError and handleSendProcessorCancel and terminate should now be synchronized as they iterate over sendingSubscriptions/channelProcessors (previously this wasn't necessary because of Cliff Click magic).